PackageInfo.g

```diff
@@ -92,13 +92,18 @@ Dependencies := rec(
 ),
 
 AvailabilityTest := function()
-        return true;
-    end,
+  if Filename(DirectoriesPackagePrograms("curlInterface"),
+              "curl.so") = fail then
+    Info(InfoWarning, 1,
```

Comment:
Hmm, shouldn't it be using `LogPackageLoadingMessage`, like in the example from the Example package? In this case it will also record this message in the package loading log.

Reply:
@mtorpey See also https://www.gap-system.org/Manuals/doc/ref/chap76.html#X7D162DDF813D2BBA and https://www.gap-system.org/Manuals/doc/ref/chap76.html#X7E4F39867CCC6026

Reply:
Thanks – amended accordingly!